Description: Decrease the priority of qt-terminal to be set as "default"
Author: Anton Gladky <gladky.anton@gmail.com>
Bug-Debian: http://bugs.debian.org/668955
Last-Update: 2012-04-16

Index: gnuplot.deb/src/term.c
===================================================================
--- gnuplot.deb.orig/src/term.c	2014-04-14 00:05:03.487277601 -0700
+++ gnuplot.deb/src/term.c	2014-04-14 00:05:03.487277601 -0700
@@ -1865,14 +1865,14 @@
 	    term_name = "sun";
 #endif /* SUN */
 
-#ifdef QTTERM
+#ifdef WXWIDGETS
 	if (term_name == (char *) NULL)
-		term_name = "qt";
+		term_name = "wxt";
 #endif
 
-#ifdef WXWIDGETS
+#ifdef QTTERM
 	if (term_name == (char *) NULL)
-		term_name = "wxt";
+		term_name = "qt";
 #endif
 
 #ifdef _Windows
